### ISSUE TITLE
Rule: duplicated settings

### ISSUE THREAD

**Issue by bhirsz** [bhirsz/robotframework-robocop#89]

**Comment by mnojek** [comment]
Is this rule only related to duplicated settings or any section?

**Comment by mnojek** [comment]
> Is this rule only related to duplicated settings or any section?

I know realized that it may be related to settings for specific section, e.g. _[Tags]_ under specific test case, right? :)

**Comment by bhirsz** [comment]
> Is this rule only related to duplicated settings or any section?

It's related to this situation:
```
Metadata  identical
Metadata  identical
Resource  common.robot
Resource common.robot
Suite Teardown  same
Suite Teardown  other but still teardown
```

Duplicated [Tags] should be detected by ParsingErrorChecker but of course we need to list such cases and test them to confirm it

**Comment by bhirsz** [comment]
Repeated settings are parsing errors:
```
Force Tag  tag
Force Tag  tag2
```
Robot Framework syntax error: Setting 'Force Tags' is allowed only once. Only the first value is used.

Permittable duplicates are Resource, Library, Metadata and Variables. 

I will only add duplicate check for those (if we're not importing the same library, with the same args twice etc)
